src/game.js

```diff
@@ -0,0 +1,115 @@
+const _ = require('lodash')
+const Betting = require('./betting')
+const currency = require('./currency')
+
+module.exports = class Game {
+  constructor() {
+    this.minBet = 10;
+    this.maxBet = 500;
+    this.CBet = 10;
+    this.minCBet = '👕'
+    this.midCBet = '👞'
+    this.maxCBet = '👖'
+    this.players = []
+    this.setup();
+    this.start();
+  }
+
+  report( theme, message ){
+    console.log(colors.green( theme+ '')+'> '+message);
+  }
+
+  prompt(ask){
+    return prompt.forString(ask)
+  }
+
+  promptForNumber(ask){
+    return prompt.forNumber(ask)
+  }
+
+  setup() {
```

Comment:
There's a lot going on in the `setup` method.  I would consider breaking it up into logical pieces that are doing one small piece of the overall setup to increase clarity (and, I suspect, to reveal opportunities to refactor).